pkg_defs/ska3-flight/meta.yaml

```diff
@@ -1,50 +1,50 @@
 ---
 package:
   name: ska3-flight
-  version: 2020.14
+  version: 2021.1
```

Comment:
Should we also just add a comment to the meta.yaml with "# For nominal releases, update ska3-core in the package list to match ska3-flight version"?  I think this is the one extra manual step that *could* be done in code but we'll probably do by hand every time.

Reply:
yes, I added a comment to that effect on ska3-core meta.yaml.